phoenix-core/src/it/java/org/apache/phoenix/end2end/UpdateCacheFreqWithDDLValidationIT.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.phoenix.end2end;
+
+import org.apache.phoenix.coprocessorclient.BaseScannerRegionObserverConstants;
+import org.apache.phoenix.query.QueryServices;
+import org.apache.phoenix.thirdparty.com.google.common.collect.Maps;
+import org.apache.phoenix.util.ReadOnlyProps;
+import org.junit.BeforeClass;
+
+import java.util.Map;
+
+//@Category(NeedsOwnMiniClusterTest.class)
+// Uncomment as part of PHOENIX-7381
```

Comment:
FYI @palashc

Reply:
When enabling last_ddl_timestamp validation, we would want to set default UCF to NEVER.

Reply:
This is not ALWAYS, this is some non default positive value e.g. 20s or 10s.

Reply:
My point is, when we uncomment this test as part of PHOENIX-7381, the tests will likely pass. Until PHOENIX-7381, these tests will not pass because even for 10/20s UCF, they still perform validate ddl timestamp and that makes getTable() call internally, hence the test fails as of now.

Reply:
Okay got it, I will look at these these tests as part of PHOENIX-7381.